### PR TITLE
add SCSS files(Description)

### PR DESCRIPTION
**Pull request by yoshino9397**
add descriptions part to develop

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./scss/main.css" />
```
Comment (Shiho317) [inline index.html]: The Styling is not valid. You need to change from `./scss/main.scss` to `./css/main.scss`.
Then I could see your styling correctly:D
```suggestion
    <link rel="stylesheet" href="./css/main.css" />
```

**Review by Shiho317 — CHANGES_REQUESTED** [review]
Hi Yoshino! Thank you for changing styling files.
There is only one little problem I found. I left as a comment at that part so pls change it.
Thank you:)

File: index.html (hunk 1 above)
Comment (yoshino9397) [reply index.html]: thanks for your review! I fixed!

**Comment by yoshino9397** [comment]
I fixed css link!

**Review by Shiho317 — APPROVED** [review]
Thank you for changing! This looks good!

**Pull request merged**